netbox/templates/extras/script_list.html

```diff
@@ -42,7 +42,7 @@ <h5 class="card-header" id="module{{ module.pk }}">
             <div class="alert alert-warning d-flex align-items-center" role="alert">
               <i class="mdi mdi-alert"></i>
               {% blocktrans trimmed with file_path=module.full_path %}
-                Script file at <code>{{ file_path }}</code> could not be loaded.
+                Script file at &nbsp;<code>{{ file_path }}</code>&nbsp; could not be loaded.
```

Comment:
If additional horizontal space is needed, we should tweak `padding` for the `<code>` element instead.

Reply:
It is just here that the spacing is off, I looked at other <code block usages and the spacing looks correct.  I thought the &nbsp; was cleaner solution then a specific css adjustment for .alert code, but can change.

Reply:
changed to use mx-1 spacing.